docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -0,0 +1,5 @@
+= Truststore Improvements
```

Comment:
Could do a better job at selling this feature. Include some of the benefits and why we did this in the first place.

Reply:
Added some language to describe why it is simpler.

Reply:
Maybe we could rephrase a bit further to put more weight on the benefits. I'm thinking something along these lines:
```
Keycloak introduces an improved truststores configuration options. The Keycloak truststore is now used across the server: for outgoing connections, mTLS, database drivers and more. It's no longer needed to configure separate truststores for individual areas. To configure the truststore, you can put your truststores files or certificates in the default `conf/truststores`, or use the new `truststore-paths` config option. For details refer to the relevant https://www.keycloak.org/server/keycloak-truststore[guide].
```

Reply:
Updated to this language.